src/components/Header.tsx

```diff
@@ -84,7 +86,9 @@ export function Header({ user }: UserProps) {
 												<div>
 													<Menu.Button className='focus:underline-2 focus:underline-none flex rounded-full bg-gray-800 text-sm'>
 														<span className='sr-only'>Open user menu</span>
-														<div className='text-base font-medium text-gray-300 hover:text-white'>Tom Cook</div>
+														<div className='text-base font-medium capitalize text-gray-300 hover:text-white'>
```

Comment:
a small thing, but let's not capitalize this here - kind of like how twitter shows user handles

Reply:
Ok. Just to make it clear, here `capitalize` doesn't mean uppercase. It means the first letter of the word is capitalized.

Reply:
Right.

Another good example is GitHub - it shows my username as ykdojo and not Ykdojo.